lib/src/base-styles/_nx-btn.scss

```diff
@@ -79,24 +79,22 @@
 }
 
 .nx-btn--error {
-  background-color: var(--nx-swatch-red-45);
-  border-color: var(--nx-swatch-red-45);
+  background-color: var(--nx-swatch-red-40);
+  border-color: transparent;
   color: var(--nx-swatch-white);
 
-  &:hover, &:focus, &:active {
-    background-color: var(--nx-swatch-red-30);
-    border-color: var(--nx-swatch-red-30);
-    color: var(--nx-swatch-white);
-  }
-
   &:focus {
-    background-color: var(--nx-swatch-red-45);
+    background-color: var(--nx-swatch-red-40);
     border-color: var(--nx-color-interactive-border-focus);
   }
 
+  &:hover {
+    background-color: var(--nx-swatch-red-35);
+  }
+
   &:active {
-    background-color: var(--nx-swatch-red-60);
-    border-color: var(--nx-swatch-red-60);
+    background-color: var(--nx-swatch-red-65);
+    border-color: var(--nx-swatch-red-65);
```

Comment:
Not necessary

Reply:
> Not necessary

Or are you interpreting the mockups as saying that an active+focused button (which is effectively any active button) should not have the blue border.  If that's the case, you should probably take off the shadow here too.

Reply:
> > Not necessary
> 
> Or are you interpreting the mockups as saying that an active+focused button (which is effectively any active button) should not have the blue border? If that's the case, you should probably take off the shadow here too.

Confirmed by @alexalyons126  in wall walk.

Reply:
> > Not necessary
> 
> Or are you interpreting the mockups as saying that an active+focused button (which is effectively any active button) should not have the blue border? If that's the case, you should probably take off the shadow here too.

Box-shadow removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/481/commits/5672f7696e727ba4d80bb51c8cc2676834b35edb

Reply:
> Box-shadow removed here: 5672f76

I'd prefer the border color be set back to transparent rather than duplicating the background color.  That way it's more consistent: border is always transparent except when the focus border is there

Reply:
> > Box-shadow removed here: [5672f76](https://github.com/sonatype/sonatype-react-shared-components/commit/5672f7696e727ba4d80bb51c8cc2676834b35edb)
> 
> I'd prefer the border color be set back to transparent rather than duplicating the background color. That way it's more consistent: border is always transparent except when the focus border is there

Agreed: https://github.com/sonatype/sonatype-react-shared-components/pull/481/commits/0b3cb58ec115d48aaf35451fed42ae14a8e958f0